packages/components/CHANGELOG-FIGMA-COMPONENTS.md

```diff
@@ -1,5 +1,32 @@
 # [HDS Product - Components](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?type=design&node-id=6790-10926&mode=design&t=Ps0aMGZ6F3z7bAJ4-0)
 
+## August 2nd, 2024
+
+`AppHeader` - Added a new navigation component to contain global and utility navigation elements.
+
+`AppSideNav` - Added a new component that shares features and functionality with the legacy `SideNav`.
+
+`Application Template` - Added a template component that provides a consistent starting point for the UI chrome.
+
+`SideNav` - **Deprecated** the legacy navigation component. It will be removed from the library once adoption of the `AppHeader` is complete.
```

Comment:
```suggestion
`SideNav` - **Deprecated** the legacy navigation component. It will be removed from the library once adoption of the `AppHeader` and `AppSideNav` is complete.
```